common/source/docs/common-polygon_fence.rst

```diff
@@ -1,64 +1,133 @@
+-  :ref:`FENCE_TOTAL<FENCE_TOTAL>` should not be changed manually and indicates the number of inclusion/exclusion fence points stored. (See :ref:`common-polygon_fence`)
+-  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = default of "ALL" is fine and will allow you to set up an additional :ref:`common-ac2_simple_geofence` , unless you want only a the inclusion/exclusion fences to be active if a home based cylindrical fence has also been enabled.
 
--  Leave the Action as "RTL or Land". Although, you can select other failsafe actions.
--  Go to the **PLAN** screen, and select ``FENCE`` from the dropdown box in the upper right part of the map display.
--  For a circular fence, simply click on the map and a circular exclusion fence is added to the list. You can change its radius with the first parameter of the item. You can also change it from an exclusion fence to a circular inclusion fence, via the item's dropdown box.
+.. note:: Polygon fence type includes the circular fences in the Inclusion/Exclusion fence list. Rover ignores altitudes, if set.
 
-   .. image:: ../../../images/exclusion-zone.png
-       :target: ../_images/exclusion-zone.png
-       :width: 500px
+If Plane, you can also:
 
+-  Set the :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>` as a breach boundary.
+-  Set :ref:`FENCE_AUTOENABLE<FENCE_AUTOENABLE>` = to allow automatic temporary (until reboot)forcing of :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 under certain vehicle conditions, such as upon arming or takeoff. A value of 0 disables this feature.
+-  :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` allows returning to the nearest RALLY point (See:), if loaded, instead of HOME.
 
-- For a polygonal fence, use the polygon tool on the left side of the map screen. Select "Draw a Polygon", and draw the boundary points with map clicks, then select if its an inclusion or exclusion fence type. It will be added to the fence point list.
 
-   .. image:: ../../../images/polygon_add_point.png
-       :target: ../_images/polygon_add_point.png
-       :width: 500px
+..  youtube:: yhNrtTERnJk
+    :width: 100%
+
 
-- Upload the fence(s) to the vehicle using "Write" button below the map. You can also read the fence list from the autopilot with the "Read" button.
 
+Copter GeoFence Tab
+-------------------
+
+Copter and Traditional Heli , when connected to Mission Planner, present a CONFIG tab called GeoFence which provides another way to access these parameters in those vehicles.
+
+.. image:: ../../../images/Fence_MPSetup.png
+    :target: ../_images/Fence_MPSetup.png
+
+Enabling the fence with an RC Channel Auxiliary Switch
+======================================================
+
+It is not necessary to set-up a switch to enable or disable the fence
+but if you wish to control the fence with a switch follow these
+steps:
+
+for firmware versions before 4.0:
+
+-  Go to the Mission Planner's Config/Tuning > Extended Tuning screen and set
```

Comment:
Is Mission Planner the only way geofences can be configured? Should documentation include other GCS options to enable this? QGC for example?

Reply:
added note...happy to add QGC specific section if someone wants to write it up....Mission Planner is usually used for examples because its the only GCS that actually tracks ArduPilot development.....QGC does not since some AP things are not in the std MAVLink releases....dont get me wrong, its a great app, I use it for mobile phone...

Reply:
Understood - I'd be happy to leave it off for the time being, unless others ask for it.